Decred Wallet/Features/TransactionDetails/TransactionDetailsViewController.swift

```diff
@@ -8,262 +8,308 @@ import UIKit
 import Dcrlibwallet
 import SafariServices
 
-class TransactionDetailsViewController: UIViewController, SFSafariViewControllerDelegate  {
-    @IBOutlet private weak var tableTransactionDetails: UITableView!    
-    @IBOutlet weak var tableView: UITableView!
-    @IBOutlet var detailsHeader: UIView!
-    @IBOutlet weak var amount: UILabel!
-    
+class TransactionDetailsViewController: UIViewController, SFSafariViewControllerDelegate {
+    @IBOutlet weak var headerView: UIView!
+    @IBOutlet weak var txTypeLabel: UILabel!
+    @IBOutlet private weak var transactionDetailsTable: SelfSizedTableView!
```

Comment:
Any particular reason for making this private? Generally, we do not want to access any `IBOutlet` from external classes, but we don't mark all `IBOutlets` private...